### DISCUSSION TITLE
What is the criteria used by Blocky for the options minTime and maxTime?

### DISCUSSION THREAD

**Discussion by alpominth** [0xERR0R/blocky#910] [General]
I'm not a Go programmer,  that's why I'm asking this.

What is the criteria used by Blocky for expiring cache entries with the option minTime?

/\ Is the amount of DNS queries made in a tracking window that makes the queries stay more time in the cache? Or something else?

**Comment by kwitsch** [comment]
The configuration of minTime and maxTime overrides the DNS response TTLs.
If they are configured and the original TTL is below min or above max time it is overwritten.

The internal cache entry is cleared if the time exceeds entry time + TTL.
